From lrodriguez@atheros.com  Thu Jan 14 17:22:32 2010
From: "Luis R. Rodriguez" <lrodriguez@atheros.com>
Date: Wed, 13 Jan 2010 21:18:15 -0500
Subject: driver-core: firmware_class: remove base.h header inclusion
To: greg@kroah.com
Cc: linux-wireless@vger.kernel.org, linux-kernel@vger.kernel.org, mcgrof@gmail.com, "Luis R. Rodriguez" <lrodriguez@atheros.com>, Johannes Berg <johannes@sipsolutions.net>
Message-ID: <1263435495-10966-1-git-send-email-lrodriguez@atheros.com>


base.h is used by base drivers for sharing internal structures.
Turns out firmware_class does not depend on it at all so remove it.

Cc: Johannes Berg <johannes@sipsolutions.net>
Signed-off-by: Luis R. Rodriguez <lrodriguez@atheros.com>
Acked-by: Marcel Holtmann <marcel@holtmann.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/firmware_class.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/base/firmware_class.c
+++ b/drivers/base/firmware_class.c
@@ -19,7 +19,6 @@
 #include <linux/kthread.h>
 #include <linux/highmem.h>
 #include <linux/firmware.h>
-#include "base.h"
 
 #define to_dev(obj) container_of(obj, struct device, kobj)
 
